src/workspace-data/data-table/uri-viewer/UriPreview.js

```diff
@@ -22,23 +22,24 @@ const styles = {
   },
 };
 
-const isImage = ({ contentType, name }) => {
-  return /^image/.test(contentType) || /\.(?:(jpe?g|png|svg|bmp))$/.test(name);
+export const isImage = ({ contentType, name }) => {
+  return /^image/.test(contentType) || /\.(jpe?g|png|svg|bmp)$/.test(name);
```

Comment:
removing the `?:` here and in the other `is*` functions changes the regex from a non-capturing group to a capturing group. Is there a good reason to make this change? See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Non-capturing_group for instance - a non-capturing group "…does not memorize the matched text, allowing for better performance…"